scenes/scene.py

```diff
@@ -0,0 +1,67 @@
+from abc import abstractmethod
+import threading
+from PIL import Image
+import os
+
+class Scene:
+	def __init__(self, **kwargs):
+		# call this last in your scene subclass inits. It will
+		# handle all the threading and data storage for you.
+		# Your prepare and perform methods can pass data through
+		# the scene_data dictionary
+
+		# each subclass should use its prepare method to prep
+		# an audio thread and image thread. You can join either
+		# of those threads in the prepare method to block
+		# starting this scene until they're ready, and/or join
+		# them in the perform method before you call super().perform()
+		# or just leave it alone to do one or both async
+		self.orchestrator = kwargs.get('orchestrator', None)
+		self.scene_data = {}		
+		self.image_thread = None
+		self.audio_thread = None
+
+		self.prepare_thread = threading.Thread(target=self.prepare)
+		self.prepare_thread.start()
+
+		pass
+
+	# Speak the sentence. Returns None
+	@abstractmethod
+	def prepare(self):
+		pass
```

Comment:
I think with newer versions of Python you do not need the pass inside of an abstract method.